Sources/FluentMongo/QueryBuilder+Options.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  QueryBuilder+Options.swift
+//  FluentMongo
+//
+//  Created by Valerio Mazzeo on 11/04/2019.
+//  Copyright © 2018 Asensei Inc. All rights reserved.
+//
+
+import Foundation
+import Fluent
+import MongoSwift
+
+public extension QueryBuilder where Database.Query == FluentMongoQuery {
+
+    /// Enables writing to temporary files. When set to true, aggregation stages can write data to the _tmp subdirectory in the dbPath directory.
+    @discardableResult
+    public func allowDiskUse(_ value: Bool = true) -> Self {
+        let previous = self.query.aggregateOptions
+        let options = AggregateOptions(
+            allowDiskUse: value,
```

Comment:
No tests for this?

Reply:
Nope. The internal storage property is private so can't check the value has been added.
Could make a test that enable that option on a query but there isn't anything in the results that would confirm that has been used or not.